plugins/Identify.jsx

```diff
@@ -304,8 +305,9 @@ class Identify extends React.Component {
             return {identifyResults: identifyResults};
         });
     };
-    onShow = (mode) => {
-        this.setState({mode: mode || 'Point'});
+    onShow = (mode, data) => {
+        this.setState({mode: mode || 'Point', exitTaskOnResultsClose: data.exitTaskOnResultsClose});
+        this.identifyPoint(data.pos);
```

Comment:
I'd encolose this within an `if (mode === "Point" && data.pos)` just for easier code readability, for the rest LGTM!

Reply:
Okay, I figured that it wasn't necessary since `this.identifyPoint` already checks if the `clickPoint` exists, but yeah, for easier code readability I get it.
Should be good now, thanks!